src/Body/anatomicalView.js

```diff
@@ -74,466 +72,644 @@ import '../App.css';
 
 class Anatomy extends React.Component {
 
-  constructor(props) {
```

Comment:
Github really messed up this diff compare. I promise all I did was adding `test-id`s. 
You can minimize `dev` version of the file and my version via https://www.browserling.com/tools/remove-all-whitespace and do a text comparison (https://en.rakko.tools/tools/7/) to see what I added.

Reply:
Sorry, my IDE's auto format kicked in...